src/neptune_prophet/impl/__init__.py

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+from __future__ import annotations
```

Comment:
Why do we need this?

Reply:
To be able to use the syntax `Type1 | Type2 ...`